build.mill

```diff
@@ -655,49 +654,19 @@ def launcherScript(
          |  JAVACMD="$$JAVA_HOME/bin/java"
          |fi
          |
-         |mill_jvm_opts=""
-         |init_mill_jvm_opts () {
-         |  if [ -z $$MILL_JVM_OPTS_PATH ] ; then
-         |    mill_jvm_opts_file=".mill-jvm-opts"
-         |  else
-         |    mill_jvm_opts_file=$$MILL_JVM_OPTS_PATH
-         |  fi
-         |
-         |  if [ -f "$$mill_jvm_opts_file" ] ; then
-         |    # We need to append a newline at the end to fix
-         |    # https://github.com/com-lihaoyi/mill/issues/2140
-         |    newline="
-         |"
-         |    mill_jvm_opts="$$(
-         |      echo "$$newline" | cat "$$mill_jvm_opts_file" - | (
-         |        while IFS= read line
-         |        do
-         |          mill_jvm_opts="$${mill_jvm_opts} $$(echo $$line | grep -v "^[[:space:]]*[#]")"
-         |        done
-         |        # we are in a sub-shell, so need to return it explicitly
-         |        echo "$${mill_jvm_opts}"
-         |      )
-         |    )"
-         |    mill_jvm_opts="$${mill_jvm_opts} -Dmill.jvm_opts_applied=true"
-         |  fi
-         |}
-         |
```

Comment:
@lefou do you remember what this stuff is? AFAIK, we currently do all `MILL_JVM_OPTS` processing in the mill client Java code, so not sure why we have to do some bash logic here

Reply:
Also the relevant integration tests seem to pass with this stuff deleted

Reply:
I think it might be remnant from back when `mill -i` ran everything within a single process, so we needed to make sure it was initialized correctly before that process started. Should be unnecessary now

Reply:
Exactly. This was prior to processing this in Java. This is a left over and should be no longer needed since we no longer run any task evaluation in the client JVM.